fvm/accounts_test.go

```diff
@@ -28,15 +28,17 @@ func createAccount(t *testing.T, vm *fvm.VirtualMachine, chain flow.Chain, ctx f
 		SetScript([]byte(createAccountTransaction)).
 		AddAuthorizer(chain.ServiceAddress())
 
-	tx := fvm.Transaction(txBody)
+	tx := fvm.Transaction(txBody, 0)
 
 	err := vm.Run(ctx, tx, ledger)
 	require.NoError(t, err)
 	require.NoError(t, tx.Err)
 
-	require.Equal(t, string(flow.EventAccountCreated), tx.Events[0].EventType.TypeID)
+	require.Equal(t, flow.EventAccountCreated, tx.Events[0].Type)
 
-	address := flow.Address(tx.Events[0].Fields[0].(cadence.Address))
+	data, err := jsoncdc.Decode(tx.Events[0].Payload)
+	require.NoError(t, err)
+	address := flow.Address(data.(cadence.Event).Fields[0].(cadence.Address))
```

Comment:
Use `flow.AccountCreatedEvent(event).Address()` instead here, and in the other cases below